src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -197,6 +200,21 @@ public override RowMapperColumnInfo[] GetOutputColumns()
                 }
                 return result;
             }
+
+            public void SaveAsOnnx(OnnxContext ctx)
+            {
+                var infos = GetOutputColumns();
```

Comment:
You are not using this infos anywhere. I am guessing it was originally meant to be used in the foreach but you decided to use _columns instead?